cmd/clusterctl/pkg/client/client.go

```diff
@@ -45,6 +45,15 @@ type GetClusterTemplateOptions struct {
 	WorkerMachineCount       int
 }
 
+// DeleteOptions carries the options supported by Delete.
+type DeleteOptions struct {
+	Kubeconfig           string
+	ForceDeleteNamespace bool
+	ForceDeleteCRD       bool
+	Namespace            string
+	Providers            []string
```

Comment:
Let's add a godoc to these

Reply:
If this is ok for you I'm doing this in a follow-up PR, so I can do the same also for all the other Options structs